Remove custom list row insets from scene views

DRadmir opened the pull request:
Eliminated the use of .listRowInsets(.assetListRowInsets) from multiple scene and component views to simplify layout and rely on default list row insets. This change improves consistency across scenes and reduces redundant styling.

Close: https://github.com/gemwalletcom/gem-ios/issues/1198

DRadmir committed:
Remove custom list row insets from scene views

Eliminated the use of .listRowInsets(.assetListRowInsets) from multiple scene and component views to simplify layout and rely on default list row insets. This change improves consistency across scenes and reduces redundant styling.

gemcoder21 commented on Features/Settings/Sources/Settings/Scenes/SettingsScene.swift:
```diff
@@ -26,14 +26,11 @@ public struct SettingsScene: View {
 
     public var body: some View {
         List {
-            Group {
```
I think settings needs to be adjusted, they look too big

DRadmir committed:
Merge branch 'main' of github.com:gemwalletcom/gem-ios into 1198-review-assetlistrowinsets-for-the-app

DRadmir committed:
Wrap settings sections in Group and set row insets

Settings sections in the list are now wrapped in a Group and have custom list row insets applied using .listRowInsets(.assetListRowInsets) for improved layout consistency.
